### ISSUE TITLE
Crash in Quadtastic v0.6.2

### ISSUE THREAD

**Issue by fluffy-critter** [25A0/Quadtastic#31]
Quadtastic crashed with the following error message:

```
//ImageEditor.lua:311: attempt to perform arithmetic on field 'x' (a nil value)
stack traceback:
	main.lua:321: in function '__sub'
	//ImageEditor.lua:311: in function 'get_resize_directions'
	//ImageEditor.lua:339: in function 'select_tool'
	//ImageEditor.lua:484: in function 'handle_input'
	//ImageEditor.lua:539: in function 'draw'
	/Quadtastic.lua:353: in function 'f'
	main.lua:224: in function 'draw'
	[string "boot.lua"]:468: in function <[string "boot.lua"]:436>
	[C]: in function 'xpcall'
	[string "boot.lua"]:569: in function <[string "boot.lua"]:564>
```

[If you can, describe what you've been doing when the error occurred]

I had deleted the quads from a group but not the group itself, then I used the magic wand tool to try to recreate them.

---
Affects: v0.6.2
Edition: osx

**Comment by 25A0** [comment]
Thanks for reporting, I'll look into this :)

**Comment by 25A0** [comment]
I wasn't able to reproduce this, but it should be fixed with f416d72.

**Comment by fluffy-critter** [comment]
Thanks! I wasn't able to reproduce it myself after the initial time it happened, so my repro steps might not have been what actually triggered it.
